Frontend/android/app/build.gradle

```diff
@@ -134,6 +134,8 @@ android {
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
         versionName "1.0"
+        missingDimensionStrategy 'react-native-camera', 'general'
+        missingDimensionStrategy 'react-native-camera', 'mlkit'
```

Comment:
Shouldn't it be either general or mlkit?

Reply:
Forgot to remove general